pyflakes/test/test_imports.py

```diff
@@ -184,9 +184,8 @@ def bar():
             def baz():
                 def fu():
                     pass
-        ''',
-                    m.RedefinedWhileUnused, m.RedefinedWhileUnused,
-                    m.UnusedImport, m.UnusedImport)
+        ''', *(m.ImportShadowedByImport, m.RedefinedWhileUnused,
```

Comment:
Eh, this isn't exactly clear or obvious. I understand why, but I'm not sure I agree with it.